src/main/assembly/dist/cfg/application.properties

```diff
@@ -10,17 +10,20 @@ daemon.http.port=20190
 #
 # 1.   Created in the drafts directory.
 # 2.   On submit: copied to stage and then that copy is moved to the submit-to directory.
-# 3a)  If archived successfully the bag in the draft-copy is deleted
+# 3a)  If archived successfully the bag in the stage is moved to submit-to
 # 3b)  If the deposit is rejected, the depositor can reopen the draft, modify it and resubmit it.
 #
-# stage and submit-to must be on a single mount to allow moving
-# stage.upload and draft must be on a single mount to allow moving
+# stage and submit-to must be on a single mount to allow moving from the first to the latter
+# stage.upload and draft must be on a single mount to allow moving zip-content into the draft bag
```

Comment:
What is `stage.upload`?

Reply:
Perhaps replace the life cycle approach with something like:

```
# drafts
#   - each user/uuid directory contains a bag + deposit.properties
# stage.upload
#   - contains temporary directories with zip content during processing of upload requests
#   - the temporary directories are prefixed with a user-id and uuid
#   - on failure the content is deleted
#   - on success the content is added as payload to a draft bag
# stage
#   - each uuid directory holds copies of draft deposits during processing a submit request
#   - on failure the copy is deleted
#   - on success the copy is moved to submit-to
# submitted
#   - each uuid directory contains a copy of a bag + deposit.properties that is ready for curation/ingest
```

Reply:
@jo-pol as discussed off-line we'll go for the following. Note that some keys need to be renamed:
```
#
# Directories for managing the deposits.
#
# - stage-zips: is used for uploads of zipped directories or complete deposits.
# - drafts: contains the draft deposits; single files are uploaded here directly, unzipped directories and deposits
#     are moved here from stage-zips.
# - stage-for-submit: the completed draft is copied here when the user submits it.
# - submit-to: the target directory to which the copy in stage-for-submit is moved for further processing.
# 
# N.B. stage-zips and drafts must be located on the same partition in order to support atomic moving. The same 
# holds for stage-for-submit and submit-to.
#
# Note that the stage-* directories only have temporary content, so the service should clear them automatically.
#
deposits.stage-zips=
deposits.drafts
deposits.stage-for-submit
deposits.submit-to


```